test/fakes/fake-grpc-error.ts

```diff
@@ -0,0 +1,18 @@
+import grpc from 'grpc'
+/**
+ * Convenience class for creating more specific Error objects to mimick grpc error responses.
```

Comment:
also super nit: `gRPC`

Reply:
does this mean we want to ```import grpc from 'gRPC'``` ?

Reply:
oh you must mean gRPC in the comment, because it doesn't know what gRPC is as an import